axoSlackBot.js

```diff
@@ -46,7 +46,7 @@ controller.setupWebserver(config.port,function(err,webserver) {
         var params = {
           grant_type: "authorization_code",
           code: code,
-          redirect_uri: config.redirectUri+ "authorizationCode",
+          redirect_uri: config.redirectUri.substring(0, config.redirectUri.indexOf("oauth")) + "authorizationCode",
```

Comment:
Why are you using a substring where you're searching for oauth?  Is this just to add a following '/'?  If so, you should be able to add the '/' before authorizationCode